sys-libs/error-standalone/error-standalone-1.0.ebuild

```diff
@@ -0,0 +1,12 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="standalone <error.h> implementation intended for musl"
+HOMEPAGE="https://hacktivis.me/git/error-standalone/"
+SRC_URI="https://hacktivis.me/releases/error-standalone/${P}.tar.gz"
```

Comment:
With regard to error-standalone itself: AFAIK, there's implementations which require a library. You've thankfully killed that requirement. But your version doesn't install a library at all which breaks compatibility unnecessarily. Add a .pc file and install an empty library?

Reply:
Also, the `error-standalone` commit (adding it) should have some detail on the background in the commit message.